src/main/java/build/buildfarm/instance/shard/Executions.java

```diff
@@ -150,4 +167,16 @@ public void insert(UnifiedJedis jedis, String name, String operationJson) {
   public void remove(UnifiedJedis jedis, String name) {
     executions.remove(jedis, name);
   }
+
+  public Operation merge(UnifiedJedis jedis, String actionKey) {
```

Comment:
```suggestion
  public @Nullable Operation merge(UnifiedJedis jedis, String actionKey) {
```
and import `@Nullable`